src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -348,7 +348,7 @@ private void UpdateCurrentConfiguration()
         private void UpdateConfiguration(T configuration)
         {
             _currentConfiguration = configuration;
-            _syncAfter = DateTimeUtil.Add(TimeProvider.GetUtcNow().DateTime, AutomaticRefreshInterval +
+            _syncAfter = DateTimeUtil.Add(TimeProvider.GetUtcNow().UtcDateTime, AutomaticRefreshInterval +
```

Comment:
Hmm, I would think in the context of getting a UTC now date time - `DateTime` and `UtcDateTime` would have the same time and zone.

Reply:
That was the error here. Utc kind of DateTimeOffset to DateTime has unspecified kind.